resources/lang/en/admin.php

```diff
@@ -7,5 +7,19 @@
     'accept' => 'Accept',
     'reject' => 'Reject',
     'show' => 'Show',
+    'room' => 'Room',
+    'bed1' => 'Bed 1',
+    'bed2' => 'Bed 2',
+    'bed3' => 'Bed 3',
```

Comment:
I'm not sure if we really require 3 almost identical *translations*.
You can use placeholders in the translations.

Eg.: `'bed' => 'Bed :bed_num'`

See: https://laravel.com/docs/5.7/localization#replacing-parameters-in-translation-strings
See: other examples in different translation files